projects/js-packages/critical-css-gen/package.json

```diff
@@ -0,0 +1,73 @@
+{
+	"private": true,
+	"type": "module",
+	"name": "@automattic/jetpack-critical-css-gen",
+	"version": "0.1.0-alpha",
+	"description": "A flexible Critical CSS Generator that supports multiple URLs and viewports, with both server-side and client-side generation capabilities.",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/critical-css-gen/#readme",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Critical Css Gen"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/critical-css-gen"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build:browser": "rollup -c",
+		"build:node": "tsc",
+		"build": "pnpm run clean && pnpm run build:browser && pnpm run build:node",
+		"clean": "rm -rf build-node/ && rm -rf build-browser/",
+		"test": "pnpm build && NODE_ENV=test NODE_PATH=./node_modules jest --forceExit --config=tests/config/jest.config.js"
+	},
+	"main": "./build-node/node.js",
+	"browser": "./build-browser/bundle.js",
+	"devDependencies": {
+		"@babel/core": "7.24.7",
+		"@babel/preset-env": "7.24.7",
+		"@babel/preset-typescript": "7.24.7",
+		"@rollup/plugin-commonjs": "26.0.1",
+		"@rollup/plugin-json": "4.1.0",
+		"@rollup/plugin-node-resolve": "13.3.0",
+		"@rollup/plugin-terser": "0.4.3",
+		"@rollup/plugin-typescript": "8.3.3",
+		"@types/clean-css": "4.2.5",
+		"@types/css-tree": "2.0.1",
+		"@types/node": "^20.4.2",
+		"express": "4.19.2",
+		"jest": "29.7.0",
+		"playwright": "1.45.1",
+		"prettier": "npm:wp-prettier@3.0.3",
+		"rollup": "2.79.1",
+		"rollup-plugin-polyfill-node": "0.13.0",
+		"source-map": "0.7.4",
+		"source-map-js": "1.2.0",
+		"tslib": "2.5.0",
+		"typescript": "5.0.4",
+		"webpack": "5.76.0",
+		"webpack-dev-middleware": "5.3.4"
+	},
+	"exports": {
+		".": {
+			"jetpack:src": "./src/node.ts",
+			"types": "./build-node/node.d.ts",
+			"browser": "./build-browser/bundle.js",
+			"import": "./build-node/node.js",
+			"require": "./build-node/node.js",
+			"default": "./build-node/node.js"
+		}
+	},
+	"dependencies": {
+		"clean-css": "^5.3.1",
+		"css-tree": "^2.3.1",
+		"playwright-core": "^1.45.1"
+	},
+	"peerDependencies": {
+		"playwright-core": "^1.45.1"
+	},
```

Comment:
Are you saying that we should make `playwright-core` optional and remove it from `dependencies`?

My understanding was that `peerDependencies` requires the project using the package to have `playwright-core` installed.

😕

Reply:
I think I got it.

I've made `playwright-core` a dev dependency (since we need it for the build process).

I kept it in `peerDependencies`, since the parent project should have `playwright-core` installed, which the built version would use.

Reply:
`playwright-core` has been made an optional dependency.